examples/angular/package.json

```diff
@@ -32,7 +32,7 @@
     "@angular-eslint/template-parser": "12.5.0",
     "@angular/cli": "~12.2.10",
     "@angular/compiler-cli": "~12.2.0",
-    "@infinumjs/eslint-config-angular-ts": "file:..",
+    "@infinumjs/eslint-config-angular-ts": "@infinumjs/eslint-config-angular-ts",
```

Comment:
I didn't know about this way to define a version. Is that something specific to how lerna does linking?

Reply:
Yes, that's the whole point of monorepo. 
Now, when we have two folders that are in monorepo setup (`packages`, `examples`) we can share packages between them. 
Lerna will symlink `@infinumjs/eslint-config-angular-ts` package source into Angular project.

Reply:
This is mistake, we should set a version here.